src/Symfony/Component/Notifier/Bridge/RocketChat/RocketChatTransport.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\RocketChat;
+
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Jeroen Spee <https://github.com/Jeroeny>
+ *
+ * @internal
+ *
+ * @experimental in 5.1
+ */
+final class RocketChatTransport extends AbstractTransport
+{
+    protected const HOST = 'rocketchat.com';
```

Comment:
Why is it protected and not private?

Reply:
Inherited from the `AbstractTransport` parent. If it were private, it would throw an error: https://3v4l.org/sKOqV